src/components/PostsList/PostsList.tsx

```diff
@@ -0,0 +1,80 @@
+import React from 'react';
+import classNames from 'classnames';
+import { Post } from '../../types/Post';
+
+type Props = {
+  posts: Post[];
+  postId: number | null;
+  onPost: (id: number | null) => void;
+};
+
+export const PostsList: React.FC<Props> = ({
+  posts,
+  postId,
+  onPost,
+}) => {
+  const handleSetOpened = (id: number) => {
+    return (postId !== id)
+      ? onPost(id)
+      : onPost(null);
+  };
+
+  return (
+    <div data-cy="PostsList">
+      <p className="title">
+        Posts:
+      </p>
+
+      <table
+        className="
```

Comment:
Multiple CSS classes are split across several lines, which is not necessary and makes the code less readable. You can just put all classes in one line.